### PR TITLE
Fix cmakelists to compile against other executables and fixing tests

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -58,18 +62,27 @@ add_library(${PROJECT_NAME}
   src/shapes.cpp
 )
 ament_target_dependencies(${PROJECT_NAME}
-    rclcpp shape_msgs
+    rclcpp
+		shape_msgs
```
Comment (LanderU) [inline CMakeLists.txt:66]: You've extra tabulations here.

**Comment by ahcorde** [comment]
ping

**Review by vmayoral — APPROVED** [review]

**Comment by vmayoral** [comment]
Linking issues when reproducing this locally:

```bash
Undefined symbols for architecture x86_64:
  "console_bridge::log(char const*, int, console_bridge::LogLevel, char const*, ...)", referenced from:
      bodies::ConvexMesh::useDimensions(shapes::Shape const*) in bodies.cpp.o
      bodies::BodyVector::setPose(unsigned int, Eigen::Transform<double, 3, 1, 0> const&) in bodies.cpp.o
      bodies::BodyVector::getBody(unsigned int) const in bodies.cpp.o
      bodies::createBodyFromShape(shapes::Shape const*) in body_operations.cpp.o
      bodies::Body* bodies::constructBodyFromMsgHelper<boost::variant<shape_msgs::msg::SolidPrimitive_<std::__1::allocator<void> >, shape_msgs::msg::Mesh_<std::__1::allocator<void> >, shape_msgs::msg::Plane_<std::__1::allocator<void> > > >(boost::variant<shape_msgs::msg::SolidPrimitive_<std::__1::allocator<void> >, shape_msgs::msg::Mesh_<std::__1::allocator<void> >, shape_msgs::msg::Plane_<std::__1::allocator<void> > > const&, geometry_msgs::msg::Pose_<std::__1::allocator<void> > const&) in body_operations.cpp.o
      bodies::Body* bodies::constructBodyFromMsgHelper<shape_msgs::msg::Mesh_<std::__1::allocator<void> > >(shape_msgs::msg::Mesh_<std::__1::allocator<void> > const&, geometry_msgs::msg::Pose_<std::__1::allocator<void> > const&) in body_operations.cpp.o
      bodies::Body* bodies::constructBodyFromMsgHelper<shape_msgs::msg::SolidPrimitive_<std::__1::allocator<void> > >(shape_msgs::msg::SolidPrimitive_<std::__1::allocator<void> > const&, geometry_msgs::msg::Pose_<std::__1::allocator<void> > const&) in body_operations.cpp.o
      ...
  "Assimp::Importer::ReadFileFromMemory(void const*, unsigned long, unsigned int, char const*)", referenced from:
      shapes::createMeshFromBinary(char const*, unsigned long, Eigen::Matrix<double, 3, 1, 0, 3, 1> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in mesh_operations.cpp.o
  "Assimp::Importer::SetPropertyInteger(char const*, int)", referenced from:
      shapes::createMeshFromBinary(char const*, unsigned long, Eigen::Matrix<double, 3, 1, 0, 3, 1> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in mesh_operations.cpp.o
  "Assimp::Importer::ApplyPostProcessing(unsigned int)", referenced from:
      shapes::createMeshFromBinary(char const*, unsigned long, Eigen::Matrix<double, 3, 1, 0, 3, 1> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in mesh_operations.cpp.o
  "Assimp::Importer::Importer()", referenced from:
      shapes::createMeshFromBinary(char const*, unsigned long, Eigen::Matrix<double, 3, 1, 0, 3, 1> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in mesh_operations.cpp.o
  "Assimp::Importer::~Importer()", referenced from:
      shapes::createMeshFromBinary(char const*, unsigned long, Eigen::Matrix<double, 3, 1, 0, 3, 1> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in mesh_operations.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [libgeometric_shapes.dylib] Error 1
make[1]: *** [CMakeFiles/geometric_shapes.dir/all] Error 2
make: *** [all] Error 2
---
Failed   <<< geometric_shapes	[ Exited with code 2 ]
```

**Comment by vmayoral** [comment]
https://github.com/AcutronicRobotics/geometric_shapes/commit/e02b97df2fa02dedf2f45cd10f3fe47cfd546b63 fixes it for OS X.

**Comment by vmayoral** [comment]
https://github.com/AcutronicRobotics/moveit2/commit/5305591eb848a82da4ee48e1963aa8e46bccab9b adds the installation for OS X CI.

**Pull request merged**